docs/monitoring.md

```diff
@@ -984,7 +984,7 @@ This is the component with the largest amount of instrumented metrics
 
 - namespace=spark.streaming
   - **note:** This applies to Spark Structured Streaming only. Conditional to a configuration
-  parameter: `spark.sql.streaming.metricsEnabled=true` (default is false) 
+  parameter: `spark.metrics.sparkStreamingSource.enabled` (default is false) 
```

Comment:
We may want to differentiate DStream and Structured Streaming. Before the change it is clear that it's for Structured Streaming (sql.streaming) which new name is not clear about that. Maybe `structuredStreamingSource` or even only require the metric name rule as `spark.metrics` as prefix and `enabled` as postfix so that it can use `sql.streaming`?